apollo-gradle-plugin/src/test/groovy/com/apollographql/android/gradle/integration/BasicAndroidSpec.groovy

```diff
@@ -148,16 +150,35 @@ class BasicAndroidSpec extends Specification {
         "return Currency.class;")
   }
 
-  def "adding generateOptional in Apollo Extension generates classes with apollographql Optional"() {
+  def "adding nullableValueType = `annotated` in Apollo Extension generates classes with Apollo Optional"() {
```

Comment:
*shouldn't generate?

Might be worth checking for the javax import in that case.